src/config.cc

```diff
@@ -1391,7 +1397,11 @@ void save( Class const & c )
     QDomElement paths = dd.createElement( "paths" );
     root.appendChild( paths );
 
-    for ( const auto & i : c.paths ) {
+    // Save all paths except the hard-code portable path,
+    // which is stored in the first element of list.
+    qsizetype pos = Config::isPortableVersion();
+
+    for ( const auto & i : c.paths.mid(pos) ) {
```

Comment:
pos should be portable path?

Reply:
`pos` would be `1` if it is portable version, thus skip the first element in the `c.paths`. I'm using QList `mid` API just for minimal code modifications. I agree that the `mid` name is a bit confusing.

> **QList<T> QList::mid(qsizetype pos, qsizetype length = -1) const**
>   Returns a sub-list which contains elements from this list, starting at position pos. If length is -1 (the default), all elements after pos are included; otherwise length elements (or all remaining elements if there are less than length elements) are included.
> Reference: https://doc.qt.io/qt-6.7/qlist.html#mid